spinalcordtoolbox/image.py

```diff
@@ -1049,7 +1052,7 @@ def change_shape(im_src, shape, im_dst=None):
         im_dst.data = im_src.data.reshape(shape, order="C")
     else:
         # image data may be a view
-        im_dst_data = im_src.data.copy().reshape(shape, order="F")
+        im_dst.data = im_src.data.copy().reshape(shape, order="F")
```

Comment:
Note that this one is a change is behaviour, but it really looked like a bug.

Reply:
Yeesh! This definitely seems like a bug to me, given the assignment to `im_dst.data` immediately preceding. 

Good catch!